integration-tests/tests/exit_syscall_payload.rs

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Rename this file: `git mv exit_syscall_payload.rs syscall_exit.rs`

The reason for this is that the name of this file will be output in the test runs. It will be left-justified. This way all the syscalls line up and it is easy to spot which syscall is failing.